Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -150,6 +136,67 @@ if ($fabricInstallerSecret){
     Write-DosMessage -Level "Information" -Message "Fabric.Installer clientSecret has been created."
 }
 
+
+# install IdPSS TODO move out to IdPSS ps1 script #######################################################################
+# Get Idpss app pool user 
+# Create log directory with read/write permissions for app pool user
+$idpssSettingsScope = "idpss"
+$idpssServiceUrl = Get-ApplicationEndpoint -appName $idpssConfigStore.appName -applicationEndpoint $idpssConfigStore.applicationEndPoint -installConfigPath $installConfigPath -scope $idpssSettingsScope -quiet $quiet
+    
+$idpssWebDeployParameters = Get-WebDeployParameters -serviceConfig $idpssConfigStore -commonConfig $commonConfigStore
+$idpssStandalonePath = ".\Fabric.IdentityProviderSearchService.zip"
+$idpssInstallerPath = "..\WebDeployPackages\Fabric.IdentityProviderSearchService.zip"
+$idpssInstallPackagePath = Get-WebDeployPackagePath -standalonePath $idpssStandalonePath -installerPath $idpssInstallerPath
+
+if($null -eq $credential)
+{
+  $idpssIISUser = Get-IISAppPoolUser -credential $credential -appName $idpssConfigStore.appName -storedIisUser $idpssConfigStore.iisUser -installConfigPath $installConfigPath -scope $idpssSettingsScope
+}
+
+$idpssInstallApplication = Publish-DosWebApplication -WebAppPackagePath $idpssInstallPackagePath `
+                      -WebDeployParameters $idpssWebDeployParameters `
+                      -AppPoolName $idpssConfigStore.appPoolName `
+                      -AppPoolCredential $idpssIISUser.credential `
+                      -AuthenticationType "Windows" `
+                      -WebDeploy
+
+$idpssAppPoolUser = $idpssIISUser.UserName
+
+# uses Discovery url, we should be able to remove since I Get-IISAppPoolUser above
+$idpssName = "IdentityProviderSearchService"
+#$idpssAppPoolUser = Find-IISAppPoolUser -applicationName $idpssName -discoveryServiceUrl $discoveryServiceUrl -noDiscoveryService $true -quiet $quiet
```

Comment:
this can probably be removed

Reply:
I left a few comments in to remind myself what code was failing and why. I will cleanup before check in.